pom.xml

```diff
@@ -610,6 +612,18 @@
         <version>${btm.version}</version>
         <scope>test</scope>
       </dependency>
+     <dependency>
+      <groupId>com.github.fakemongo</groupId>
+      <artifactId>fongo</artifactId>
+      <version>${fongo.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.hamcrest</groupId>
+        <artifactId>hamcrest-all</artifactId>
```

Comment:
I'm not sure about this dependency. Why do you need it? If it's for assertions on tests, remember that we are using AsssertJ for that.

Reply:
I fixed to remove this dependency.
https://github.com/Activiti/activiti-build/pull/3/commits/04fc5aa3c658a98a545af79042cbf210d0a01392